pkg/skaffold/debug/debugger_mux.go

```diff
@@ -14,17 +14,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1
+package debug
 
-import (
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/config"
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/debugging"
-)
+import "context"
 
-func (r *SkaffoldRunner) createContainerManager() *debugging.ContainerManager {
-	if r.runCtx.Mode() != config.RunModes.Debug {
-		return nil
+type DebuggerMux []Debugger
+
+func (d DebuggerMux) Start(ctx context.Context, namespaces []string) error {
+	for _, debugger := range d {
+		if err := debugger.Start(ctx, namespaces); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (d DebuggerMux) Stop() {
+	for _, debugger := range d {
+		debugger.Stop()
 	}
+}
 
-	return debugging.NewContainerManager(r.podSelector)
+func (d DebuggerMux) Name() string {
+	var name string
+	for _, debugger := range d {
+		name += debugger.Name()
```

Comment:
will this show up weirdly in the IDE  😅. Also can't we just name this as constant `DebuggerMux` or something?

Reply:
yeah that's probably fine. this is only used in a debug-level log anyway